custom-hooks/Exercise1/solution/src/App.tsx

```diff
@@ -0,0 +1,26 @@
+import './App.css'
+import Toggle from './Toggle'
+import { useToggle } from './useToggle';
+
+function App() {
+  const [on, toggle] = useToggle(true);
+
+  return (
+      <section>
+        <h1>UseToggle</h1>
+        <button disabled={on} onClick={() => toggle(true)}>
+          Turn On
+        </button>
+        <button disabled={!on} onClick={() => toggle(false)}>
```

Comment:
I would [avoid disabling buttons](https://bitovi.slack.com/archives/C055KNS4ARL/p1697473839079379?thread_ts=1697473446.424119&cid=C055KNS4ARL) if you can.